whitelist/README.md

```diff
@@ -0,0 +1,75 @@
+# Whitelist contract for staking pools
+
+The purpose of this contract is to maintain the whitelist of the staking pool contracts account IDs that are approved
+by NEAR Foundation.
+
+In order for the lockup contracts to be able delegate to a staking pool, the staking pool should provide spec guarantees.
+The staking pool should guarantee that the delegated tokens can't be lost or locked, such as the lockup contract should be
+able to recover delegated tokens back to the lockup from a staking pool. In order to enforce this, only the whitelisted
+staking pool contracts and accounts can receive delegated tokens.
```

Comment:
> staking pool contracts and accounts

This seems confusing.  Aren't the contracts the same as the account associated with the contract?

Reply:
Updated